lib/recording/RecordingManager.js

```diff
@@ -480,6 +484,56 @@ class RecordingManager {
             });
     }
 
+    handleNewLiveClippingTask(task)
+    {
+        logger.debug("TASK - in the recordingManager call back for task: " + util.inspect(task));
+        let playListPath = path.join(persistenceFormat.getEntryBasePath(task.entryId), persistenceFormat.getMasterManifestName());
+        let newPath = persistenceFormat.getRecordingSessionPath(task.entryId, null, task.clippedEntryId);
+        let newPathInLive = persistenceFormat.getEntryBasePath(task.clippedEntryId);
+        let duration = task.clipAttributes.duration;
+        let strDuration = duration.toString();
+        let flavors = null;
+
+        return qio.makeTree(newPath).then(() => {
+            return qio.read(playListPath);
+        }).then((data) => {
+            let playList = JSON.parse(data);
+            playList.clipFrom = PlaylistGenerator.getAbsoluteTimeFromRelative(playList, task.clipAttributes.offset);
+            playList.clipTo = PlaylistGenerator.getAbsoluteTimeFromRelative(playList, task.clipAttributes.offset + duration);
+            if (playList.clipFrom >= playList.clipTo)
+            {
+                logger.error("VALIDATION ERROR: can not set playlist attribute: clipFrom: [" +playList.clipFrom + "] and clipTo: [" + playList.clipTo + "]");
+                backendClient.updateEntryServerNodeStatus(task.id, kalturaTypes.KalturaEntryServerNodeStatus.ERROR);
```

Comment:
@david-winder-kaltura 
i think the updateEntryServerNodeStatus should go in the catch that ends the function below  to avoid code duplication